components/formats-gpl/src/loci/formats/in/CellWorxReader.java

```diff
@@ -711,14 +714,16 @@ private IFormatReader getReader(String file)
       pnl = new MetamorphReader();
     }
 
-    IMetadata metadata;
-    try{
-      metadata = service.createOMEXMLMetadata();
-    }
-    catch (ServiceException exc) {
-      throw new FormatException("Could not create OME-XML store.", exc);
+    if (omexml) {
+      IMetadata metadata;
+      try {
```

Comment:
I might well be incorrect here, but isn't ome-xml.jar now a hard dependency here since the OMEXMLServiceImpl implementation is in formats-api as well as the interface?